x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/cards/protection_seetings_card/components/reputation_service.tsx

```diff
@@ -104,7 +104,7 @@ export const ReputationService = React.memo(
                 content={
                   <FormattedMessage
                     id="xpack.securitySolution.endpoint.policyDetailsConfig.reputationServiceTooltip"
-                    defaultMessage="This option enables/disables the Reputation Service feature in Endpoint. When the option is ON, Endpoint will reach out to a Cloud API for additional malware analysis. When it's OFF, Endpoint will not reach out to the Cloud API." // TODO: update message
+                    defaultMessage="This option enables/disables the Reputation Service feature in Endpoint. When the option is ON, Endpoint will reach out to a Cloud API for additional detection coverage. When it's OFF, Endpoint will not reach out to the Cloud API, resulting in reduced efficacy."
```

Comment:
> resulting in reduced efficacy

I used this slightly-stronger working instead of "may result in reduced efficacy" taking "efficacy" as the totality of our protections.  It is a 100% true statement that "We are less effective when this feature is off."

Reply:
➕